hw/ip/flash_ctrl/data/flash_ctrl.hjson

```diff
@@ -551,7 +551,88 @@
       default:   "5",
     },
   ],
+  features: [
+    {
+     name: "FLASH_CTRL.ESCALATION"
+     desc:'''Flash controller has two sources of escalation.
+     1. Global escalation : This is initiated by the LC (Lifecycle Controller) through the use of lc_escalate_en.
+     2. Local escalation : This is activated in response to standard faults detected within the flash.
+                           These faults are monitored through flash_ctrl.STD_FAULT_STATUS.
+     '''
+    }
+    {
+      name: "FLASH_CTRL.FETCH_CODE"
+      desc: '''If SW program flash_ctrl.EXEC to 0xA26A38F7, code fetch from flash device is allowed.
+            All test_rom test uses this features. No dedicated test is required.
+      '''
+    }
+    {
+      name: "FLASH_CTRL.INFO.CREATOR_PARTITION"
+      desc: '''This partition stores creator root seed.
+            It is accessed by the flash controller after scramble key is received.
+            SW can read or program the contents with both conditions are satisfied.
+              1. Life cycle state should be either one of DEV / PROD / PROD_END / RMA.
+              2. The device has not yet been personalized.
```

Comment:
meaning secret2 OTP partition has not yet been written and locked? (maybe be more specific)